src/server/injected/injectedScript.ts

```diff
@@ -662,7 +662,7 @@ export class InjectedScript {
 
   isElementDisabled(element: Element): boolean {
     const elementOrButton = element.closest('button, [role=button]') || element;
-    return ['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA'].includes(elementOrButton.nodeName) && elementOrButton.hasAttribute('disabled');
+    return ['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA'].includes(elementOrButton.nodeName) && elementOrButton.hasAttribute('disabled') || elementOrButton.getAttribute('aria-disabled') === 'true';
```

Comment:
- Let's not check `aria-disabled` for buttons, inputs, selects and textareas. AFAIU, they have no effect there.
- From my testing on Chromium, everything but "false", "FALSE" and other case variations is considered "disabled". Let's change the check accordingly.

Reply:
I don't check aria-disabled for buttons. There is `||`. Or what do you mean?

Reply:
If you have `<button aria-disabled='true'>`, this will return `true` (because second branch is true). I believe that browsers will ignore aria-disabled in this case though, and the button will be considered enabled.

Reply:
aria never effects the behavior of the page, only the behavior of screen readers reading the page. So a button with aria-disabled=true will be clickable, but the screen reader will report it as not clickable.
![image](https://user-images.githubusercontent.com/4624233/106250342-59a0dc80-61c8-11eb-9f9f-fc10b57ffc84.png)